packages/lib/holidays/constants.ts

```diff
@@ -0,0 +1,271 @@
+  TW: { name: "Taiwan", calendarId: "en.taiwan.official#holiday@group.v.calendar.google.com" },
+  TJ: { name: "Tajikistan", calendarId: "en.tj.official#holiday@group.v.calendar.google.com" },
+  TZ: { name: "Tanzania", calendarId: "en.tz.official#holiday@group.v.calendar.google.com" },
+  TH: { name: "Thailand", calendarId: "en.th.official#holiday@group.v.calendar.google.com" },
+  TL: { name: "Timor-Leste", calendarId: "en.tl.official#holiday@group.v.calendar.google.com" },
+  TG: { name: "Togo", calendarId: "en.tg.official#holiday@group.v.calendar.google.com" },
+  TO: { name: "Tonga", calendarId: "en.to.official#holiday@group.v.calendar.google.com" },
+  TT: { name: "Trinidad & Tobago", calendarId: "en.tt.official#holiday@group.v.calendar.google.com" },
+  TN: { name: "Tunisia", calendarId: "en.tn.official#holiday@group.v.calendar.google.com" },
+  TR: { name: "Turkey", calendarId: "en.turkish.official#holiday@group.v.calendar.google.com" },
+  TM: { name: "Turkmenistan", calendarId: "en.tm.official#holiday@group.v.calendar.google.com" },
+  TC: { name: "Turks & Caicos Islands", calendarId: "en.tc.official#holiday@group.v.calendar.google.com" },
+  TV: { name: "Tuvalu", calendarId: "en.tv.official#holiday@group.v.calendar.google.com" },
+  VI: { name: "U.S. Virgin Islands", calendarId: "en.vi.official#holiday@group.v.calendar.google.com" },
+  UG: { name: "Uganda", calendarId: "en.ug.official#holiday@group.v.calendar.google.com" },
+  UA: { name: "Ukraine", calendarId: "en.ukrainian.official#holiday@group.v.calendar.google.com" },
+  AE: { name: "United Arab Emirates", calendarId: "en.ae.official#holiday@group.v.calendar.google.com" },
+  GB: { name: "United Kingdom", calendarId: "en.uk.official#holiday@group.v.calendar.google.com" },
+  US: { name: "United States", calendarId: "en.usa.official#holiday@group.v.calendar.google.com" },
+  UY: { name: "Uruguay", calendarId: "en.uy.official#holiday@group.v.calendar.google.com" },
+  UZ: { name: "Uzbekistan", calendarId: "en.uz.official#holiday@group.v.calendar.google.com" },
+  VU: { name: "Vanuatu", calendarId: "en.vu.official#holiday@group.v.calendar.google.com" },
+  VA: { name: "Vatican City", calendarId: "en.va.official#holiday@group.v.calendar.google.com" },
+  VE: { name: "Venezuela", calendarId: "en.ve.official#holiday@group.v.calendar.google.com" },
+  VN: { name: "Vietnam", calendarId: "en.vietnamese.official#holiday@group.v.calendar.google.com" },
+  WF: { name: "Wallis & Futuna", calendarId: "en.wf.official#holiday@group.v.calendar.google.com" },
+  YE: { name: "Yemen", calendarId: "en.ye.official#holiday@group.v.calendar.google.com" },
+  ZM: { name: "Zambia", calendarId: "en.zm.official#holiday@group.v.calendar.google.com" },
+  ZW: { name: "Zimbabwe", calendarId: "en.zw.official#holiday@group.v.calendar.google.com" },
+};
+
+/** Get holiday cache duration from env or use default */
+export function getHolidayCacheDays(): number {
```

Comment:
we may not need a functionality to control this with env var. a const value might be enough until we get a demand.